client/components/data/query-user-purchases/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
```

Comment:
Same as above. `React` needs to be always exposed explicitly when JSX is involved.